### PR TITLE
feat: foxpage header and tabs responsive

### PR DESCRIPTION
**Pull request by NeOMakinG**
## Description
- As we now have responsive mockups, we need to adapt the work we did to new mockups
- I switched the tab behavior to a manual one
- I used a `Menu` component including some tabs too
- I changed the `FoxTab` mobile style
- Fixed some wrong padding added to app code
- Added this responsive behavior to app code

The selected asset logic should be done in application code and after the architecture planning so we know what to do, but at least the tab logic works inside the storybook story

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk
#1766 #1765 
- No risk for the moment, inside storybook stories

## Testing
- Go to `FoxPage` in the app, and see if it looks fine on desktop
- Use the mobile tool of storybook to check on small, large mobile
image
- See if the component is a dropdown and if selecting an asset switch the `TabPanels` content
- I also removed the background of the header component on mobile as it's not needed anymore

## Screenshots (if applicable)
image
image

### CODE REVIEW

File: src/plugins/foxPage/components/Layout.stories.tsx [hunk 1]
```diff
@@ -23,6 +42,9 @@ const mockAsset = {
 
 export const FoxLayout: Story = () => {
   const translate = useTranslate()
+  const [tabIndex, setTabIndex] = useState(0)
```
Comment (gomesalexandre) [inline src/plugins/foxPage/components/Layout.stories.tsx:45]:
```suggestion
  const [selectedTabIndex, setSelectedTabIndex] = useState(0)
```

**Review by gomesalexandre — COMMENTED** [review]
Overall LGTM, but noticed that with the mobile version, selecting an asset doesn't actually select it (this can be verified by reopening the menu).

The PR description states that
> The selected asset logic should be done in application code and after the architecture planning so we know what to do, but at least the tab logic works inside the storybook story

But it's not exactly true, the logic is actually logic related to the tabs component itself and should be added on the `!isLargerThanMd` branch to not only set the index, but also be in selected state according to the current item index (not) matching the current index.

Given Chakra UI's Menu (https://chakra-ui.com/docs/components/overlay/menu) isn't really made to have such selected state and the first item will always be selected, this is something you'll probably have to implement manually with a mix of `isSelected` style logic and custom styling

**Comment by NeOMakinG** [comment]
I've a doubt about the current implementation of the `foxPage` component

I did a small refactor here: https://github.com/shapeshift/web/commit/4986e6aa4f973fdc80286263cf02979b2ddc1992

If someone feels like it's better, feel free to say it, it would permit us to extract a lot of logic to a `Tabs` component

**Review by gomesalexandre — APPROVED** [review]
Pulled this one locally and responsive LGTM. refactor PR fixes almost all of the concerns I had on this one.

Still the same issue about the asset with selected state in the dropdown always being FOX, but that can be tackled in the refactor/in a follow-up as we're currently too tightly coupled with the tabs logic here

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**